global/andromeda/templates/deployment.yaml

```diff
@@ -62,6 +62,18 @@ spec:
                 secretKeyRef:
                   name: sentry
                   key: {{ include "andromeda.fullname" . }}.DSN
+            - name: OS_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: andromeda-secret
+                  key: service_user_password
+            {{- if .Values.audit.enabled }}
+            - name: AUDIT_TRANSPORT_URL
```

Comment:
`AUDIT_TRANSPORT_URL` env is not yet supported by andromeda, I've created a PR for that: https://github.com/sapcc/andromeda/pull/722